marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/model/CreateAndUpdateModelTest.java

```diff
@@ -69,6 +69,7 @@ void testModelsServicesEndpoints() {
     void testCreateNamespacedModel() {
         runAsTestUserWithRoles("hub-central-entity-model-writer");
         createNamespacedModel();
+        updateHubCentral();
```

Comment:
Hmm - in this context, I think we really need a qualifier on "HubCentral" because this sounds like "I'm going to update Hub Central the application". I think "hubCentral" is totally fine in the model definition itself, where it's obvious that it's configuration. 

Since we're only focused on adding graph config, how about calling the data that you're creating/updating/verifying "GraphConfig" - so "updateModelWithGraphConfig"?